### PR TITLE
Chore/ts data explorer

### PR DESCRIPTION
**Pull request by 121watts**
Closes #

_Briefly describe your proposed changes:_
_What was the problem?_
_What was the solution?_

  - [ ] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [ ] Rebased/mergeable
  - [ ] Tests pass
  - [ ] swagger.json updated (if modified Go structs or API)
  - [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/data_explorer/reducers/ui.ts [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+  type: 'DE_DELETE_QUERY'
+  payload: {
+    queryID: string
+  }
+}
+
+type Action = ActionAddQuery | ActionDeleteQuery
+
 const initialState = {
   queryIDs: [],
```
Comment (ebb-tide) [inline ui/src/data_explorer/reducers/ui.ts]: TS can't infer type here? is it just an empty array?

File: ui/src/ifql/components/TimeMachine.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import TimeMachineVis from 'src/ifql/components/TimeMachineVis'
 import Threesizer from 'src/shared/components/Threesizer'
 import {Suggestion, OnChangeScript, FlatBody} from 'src/types/ifql'
 import {ErrorHandling} from 'src/shared/decorators/errors'
-import {HANDLE_VERTICAL, HANDLE_HORIZONTAL} from 'src/shared/constants/index'
+import {HANDLE_VERTICAL, HANDLE_HORIZONTAL} from 'src/shared/constants'
```
Comment (ebb-tide) [inline ui/src/ifql/components/TimeMachine.tsx]: in this file line 61, visibility is not typed

File: ui/src/shared/actions/timeSeries.ts [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
 
 export const handleLoading = (query, editQueryStatus) => {
-  editQueryStatus(query.id, {loading: true})
+  editQueryStatus(query.id, {
+    loading: true,
+  })
 }
+
 // {results: [{}]}
 export const handleSuccess = (data, query, editQueryStatus) => {
```
Comment (ebb-tide) [inline ui/src/shared/actions/timeSeries.ts]: These are not typed but I think these are also outside the scope of data explorer.

**Comment by 121watts** [comment]
Great eye Deniz!  Pushed up some changes.  😄
